package/template-cartesian-graph/package

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(template-cartesian-graph)
+pkgdesc="A Cartesian graph template for the reMarkable tablet"
+url=https://github.com/PeterGrace/template-cartesian-graph
+pkgver=1.0.0-4
+timestamp=2021-01-14T17:46Z
+section=utils
+maintainer="Peter Grace <pete.grace@gmail.com>"
+license=MIT
+depends=(templatectl)
+
+image=qt:v1.2.2
+source=(https://github.com/PeterGrace/template-cartesian-graph/archive/v1.0.0.zip)
+sha256sums=(d3215546c6fd709bb5988b7042ce5fb990d9a030d783ed7505fcc2dbedbc0e0f)
+
+build() {
+    echo "no build steps at this time"
+}
+
+package() {
+    install -D -m 755 -t "$pkgdir"/opt/share/remarkable/templates \
+        "$srcdir"/template-cartesian-graph.png
+    install -D -m 755 -t "$pkgdir"/opt/share/remarkable/templates \
+        "$srcdir"/template-cartesian-graph.svg
+}
+
+configure() {
+    o_file="/opt/share/remarkable/templates/template-cartesian-graph"
+    t_file="/usr/share/remarkable/templates/template-cartesian-graph"
+    for ext in png svg; do
+        if [[ -L "${pkgdir}${t_file}.${ext}" ]]; then
+            if [[ "$(readlink \"${pkgdir}${t_file}.${ext}\")" != "${pkgdir}${o_file}.${ext}" ]]; then
```

Comment:
A bit counterintuitive but I don’t think the inner double quotes need escaping here.
```suggestion
            if [[ "$(readlink "${pkgdir}${t_file}.${ext}")" != "${pkgdir}${o_file}.${ext}" ]]; then
```